05_classify_mutations_multimodal/compare_mlp.ipynb

```diff
@@ -1,5 +1,14 @@
 {
```

Comment:
<p>What is the difference between <code>mlp, random</code> and <code>mlp, random, 5000 PCs</code>? It seems like only using PCs seems to really tank the MLP performance, but maybe that setting makes it more comparable to the elastic nets?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/mpmp/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p><code>mlp, random</code> in the old results was with 1000 PCs. We ended up not using those results because like you said it's harder to make the case that it's exactly comparable to the elastic net setup (those models use 5000 PCs), although the performance is generally better, presumably because the 5000 PC NNs end up overfitting somewhat.</p><p> </p><p>It didn't change the comparative results (in <code>05_classify_mutations_multimodal/plot_multimodal_results.ipynb</code>)to use 1000 PCs, so we decided not to explore it any further, even though the individual performance for 1000 PCs was more comparable to the elastic net models.</p>